Makefile

```diff
@@ -44,3 +45,29 @@ local-k8s-uninstall:
 	kubectl delete -f $(DEPLOY_FOLDER)/csi-nfs-driverinfo.yaml --ignore-not-found
 	kubectl delete -f $(DEPLOY_FOLDER)/rbac-csi-nfs-controller.yaml --ignore-not-found
 	echo "Uninstalled NFS driver"
+
+
+.PHONY: kind-install-driver
+kind-install-driver:
+	echo "Instlling locally"
+	kubectl apply -f $(DEPLOY_FOLDER)/rbac-csi-nfs-controller.yaml
+	kubectl apply -f $(DEPLOY_FOLDER)/csi-nfs-driverinfo.yaml
+	kubectl apply -f ./test/e2e/manifests/csi-nfs-controller.yaml
+	kubectl apply -f ./test/e2e/manifests/csi-nfs-node.yaml
+	echo "Successfully installed"
+
+.PHONY: setup-external-e2e
+setup-external-e2e:
+	curl -sL https://storage.googleapis.com/kubernetes-release/release/v1.14.0/kubernetes-test-linux-amd64.tar.gz --output e2e-tests.tar.gz
+	tar -xvf e2e-tests.tar.gz
+	rm e2e-tests.tar.gz
+	mkdir /tmp/csi-nfs
+	cp ./kubernetes/test/bin/e2e.test /tmp/csi-nfs/e2e.test
+	rm -r kubernetes
+	cp ./examples/kubernetes/storageclass-nfs.yaml /tmp/csi-nfs/storageclass.yaml
```

Comment:
is it possible to use `./examples/kubernetes/storageclass-nfs.yaml` file directly?

Reply:
I did try, but I was facing some issues with the relative path. Not really sure what went wrong, but I will debug